### PR TITLE
added 1 folder and 1 file to it. added info for about page

### PR DESCRIPTION
**Pull request by Krikfrik**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <h1>Intellect Fox</h1>
+        <h3>Front-end 4</h3>
+        <hr/>
+        <h5>Почему я выбрала направление Front-end разработки &#10067</h5>
```
Comment (ghost) [inline about/index.html:9]: Tag `head` shouldn't include `h1, h3, hr, h5` tags.  That tag only for metadata for the browser. Please move that to `body` tag.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <body>
+        <p>Я очень долго порывалась начать обучение в сфере IT (<del>а хочу я туда конечно же ради денег &#128176</del>), но не знала, с какой сторона подступиться, так как вообще не имею опыта в программировании.
+        <br/>На сайте курса говорится:
+            <blockquote cite="https://intellectfox.by/">
+                <strong>Для того, чтобы <em>«войти»</em> в IT как инженер, профессия Front-end разработчика — отличный вариант для тех,
+                    <br/>кто никогда не занимался программированием. Технологии, которые необходимы для этой профессии, 
+                    <br>достаточно легко освоить и язык JavaScript — один из самых простых в программировании.</strong>
+            </blockquote>
+            Из чего я поняла, что данное направление - отличное начало пути. &#9786
+        </p>
```
Comment (ghost) [inline about/index.html]: It's bad practice to separate `p` tag with help `br`.  If you want to get some text by the new line, create another tag `p`.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <hr/>
+        <p> <span><ins>Имя ученицы:</ins> Анна Гирковская</span>
+            <br/>
+            <span><ins>Ссылки:</ins> <a href="https://github.com/Krikfrik">GitHub</a>, <a href="mailto:bennaginger21@gmail.com">Почта</a></span>
+        </p>
```
Comment (ghost) [inline about/index.html]: See the previous comment.

File: about/index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <br/>
+        <br/>
+        <p><span>вот вам химическая формула серотонина <del>вдруг вам тоже его не хватает в последнее время</del></span>
+            <br>
+            <mark>C<sub>10</sub>H<sub>12</sub>N<sub>2</sub>O</mark>
+        </p>
```
Comment (ghost) [inline about/index.html]: See the previous comment.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+        <h1>Моя первая страница</h1>
+    </head>
+    <body>
+        <p><a href="/index.html">Об авторе</a></p>
```
Comment (ghost) [inline index.html]: Seems you should write  `<p><a href="./about/index.html">Об авторе</a></p>`

File: about/index.html (hunk 1 above)
Comment (Krikfrik) [reply about/index.html:9]: Moved to body tag.

File: about/index.html (hunk 2 above)
Comment (Krikfrik) [reply about/index.html]: created another tags p.

File: about/index.html (hunk 3 above)
Comment (Krikfrik) [reply about/index.html]: removed tag p

File: about/index.html (hunk 4 above)
Comment (Krikfrik) [reply about/index.html]: removed tag p

File: index.html (hunk 5 above)
Comment (Krikfrik) [reply index.html]: changed to <p><a href="./about/index.html">Об авторе</a></p>

**Pull request merged**